public/dorequest.php

```diff
@@ -137,18 +137,21 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
     /*
      * Global, no permissions required
      */
+
+    // TODO deprecate?
     case "allprogress":
         $consoleID = (int) request()->input('c');
         $response['Response'] = GetAllUserProgress($username, $consoleID);
         break;
 
+    // TODO deprecate - next image ID should be provided by the server
```

Comment:
This is used by the DLL to determine the maximum value allowed for manually specifying a badge.

We could remove the validation and just allow the user to enter any number, but I'm not sure how well the client would handle the 404 response.